src/rtnl/constants.rs

```diff
@@ -463,6 +463,22 @@ pub const IFLA_BR_VLAN_STATS_PER_PORT: u16 = 45;
 pub const IFLA_BR_MULTI_BOOLOPT: u16 = 46;
 pub const IFLA_BR_MCAST_QUERIER_STATE: u16 = 47;
 
+pub const IFLA_MACSEC_UNSPEC: u16 = 0;
+pub const IFLA_MACSEC_SCI: u16 = 1;
+pub const IFLA_MACSEC_PORT: u16 = 2;
+pub const IFLA_MACSEC_ICV_LEN: u16 = 3;
+pub const IFLA_MACSEC_CIPHER_SUITE: u16 = 4;
+pub const IFLA_MACSEC_WINDOW: u16 = 5;
+pub const IFLA_MACSEC_ENCODING_SA: u16 = 6;
+pub const IFLA_MACSEC_ENCRYPT: u16 = 7;
+pub const IFLA_MACSEC_PROTECT: u16 = 8;
+pub const IFLA_MACSEC_INC_SCI: u16 = 9;
+pub const IFLA_MACSEC_ES: u16 = 10;
+pub const IFLA_MACSEC_SCB: u16 = 11;
+pub const IFLA_MACSEC_REPLAY_PROTECT: u16 = 12;
+pub const IFLA_MACSEC_VALIDATION: u16 = 13;
+pub const IFLA_MACSEC_PAD: u16 = 14;
```

Comment:
If `IFLA_MACSEC_PAD` is not used, please comment it out or remove it.

Reply:
Done!